JM/CF/Debugger/3_Game/CF_Debugger/CF_Debugger.c

```diff
@@ -0,0 +1,199 @@
+/* Example:
+
+void EOnSimulate(...)
+{
+	auto dbg_Helicopters = CF.Debugger.Get("Helicopters", this);
+
+	dbg_Helicopters.Add("Position", GetPosition());
+	dbg_Helicopters.Add("Orientation", GetOrientation());
+
+	CF.Debugger.Display("Helicopters", this, "Velocity", GetVelocity(this));
+}
+
+*/
+
+class CF_Debugger
+{
+	private autoptr array<ref CF_Debugger_Block> m_Blocks;
+	private bool m_IsEnabled;
+
+	void CF_Debugger()
+	{
+		m_Blocks = new array<ref CF_Debugger_Block>();
+
+		m_IsEnabled = FileExist("$profile:CF_Debugger.txt");
```

Comment:
Why does it need this file? There should be a way to enable and disable this purely via script and or startup parameter?

e.g. DayZ.exe -CF_DebugUI